tonic-health/src/lib.rs

```diff
@@ -28,6 +28,10 @@ pub mod proto {
     #![allow(unreachable_pub)]
     #![allow(missing_docs)]
     tonic::include_proto!("grpc.health.v1");
+
+    #[allow(dead_code)]
+    pub const GRPC_HEALTH_V1_FILE_DESCRIPTOR_SET: &'static [u8] =
```

Comment:
Would it make sense to include a test here maybe?

Reply:
I can do that. It will be a few days before I have the time, but I will get that worked in.